src/cascadia/WindowsTerminal/AppHost.cpp

```diff
@@ -98,31 +71,36 @@ AppHost::~AppHost()
 // - <none>
 // Return Value:
 // - <none>
-void AppHost::_handleCommandlineArgs()
+void AppHost::_HandleCommandlineArgs()
 {
     if (auto commandline{ GetCommandLineW() })
     {
         int argc = 0;
 
         // Get the argv, and turn them into a hstring array to pass to the app.
         LPWSTR* argv = CommandLineToArgvW(commandline, &argc);
-        if (argc > 0)
+        auto deleter = wil::scope_exit([&]() { LocalFree(argv); });

```

Comment:
Now what you've got here is a `wil::unique_hlocal_string` :smile:

Reply:
No I actually don't - `wil::unique_hlocal_string` is a `PWSTR`. I've got a `PWSTR*`, so I need a `wil::unique_any<LPWSTR*, decltype(&::LocalFree), ::LocalFree>` 🤢